File: src-model/model/celestials/types/Planet.java
package model.celestials.types;

import model.celestials.CelestialBody;
import model.celestials.Orbit;
import model.celestials.Sphere;

public class Planet extends CelestialBody {

    public Planet(String name, Orbit orbit, Sphere sphere) {
        super(name, orbit, sphere);
    }
}

File: src/game/model/celestials/Planet.kt
package game.model.celestials

import game.model.celestials.parts.Clouds
import game.model.celestials.parts.Orbit
import game.model.celestials.parts.Ring
import game.model.celestials.parts.Sphere

class Planet(
        name: String,
        orbit: Orbit,
        clouds: Clouds,
        ring: Ring,
        sphere: Sphere
) : CelestialBody(name, orbit, sphere, clouds, ring)
